models/articles.model.js

```diff
@@ -72,3 +72,31 @@ exports.updateArticleById = (inc_votes, article_id) => {
       }
     });
 };
+
+exports.fetchCommentsByArticleId = (article_id) => {
+  return db
+    .query(
+      `SELECT comment_id, votes, created_at, author, body 
+    FROM comments
+    WHERE article_id = $1 ORDER BY created_at DESC;`,
+      [article_id]
+    )
+    .then((comments) => {
+      if (comments.rows.length !== 0) {
+        return comments.rows;
+      } else {
+        return db
+          .query(`SELECT * FROM articles WHERE article_id = $1`, [article_id])
+          .then((response) => {
```

Comment:
We have a nested then block here, you could refactor this using a Promise.all() to avoid this